gas/testsuite/gas/riscv/cv-postinc-fail-march.d

```diff
@@ -0,0 +1,3 @@
+#as:
```

Comment:
This is fine, however, maybe for consistency with other tests use '#as: -march=rv32i'.
This is testing for absence of xcorev/xcorevpostinc so there is no problem either way, just for consistency.